h/static/scripts/test/drafts-test.js

```diff
@@ -54,9 +54,9 @@ describe('drafts', function () {
 
   describe('#unsaved', function () {
     it('should return drafts for unsaved annotations', function () {
-      var model = {};
+      var model = {$$tag: 'local-tag', id: undefined};
       drafts.update(model, {text: 'bar'});
-      assert.deepEqual(drafts.unsaved(), [model]);
+      assert.match(drafts.unsaved(), sinon.match([model]));
```

Comment:
This is `sinon.assert.match`, it lets you check a subset of the shape of a returned object:

```js
assert.deepEqual({a: 42, dontCare: false}, {a: 42}) // throws
sinon.assert.match({a: 42, dontCare: false}, {a: 42}) // OK
```